themes/bootstrap3/templates/Recommend/TopFacets.phtml

```diff
@@ -12,7 +12,7 @@
     <div class="top-facets">
       <!--Toggle for mobile collapse -->
       <?php $topLabel = $this->transEsc('top_facet_label', ['%%label%%' => $this->translate($cluster['label'])]); ?>
-      <a class="top-title js-toggle visible-xs" data-toggle="collapse" data-target="#top_<?=$this->escapeHtml($title) ?>"><i class="fa fa-caret-right"></i> <?=$topLabel ?></a>
+      <a class="top-title js-toggle visible-xs" data-toggle="collapse" data-target="#top_<?=$this->escapeHtml($title) ?>"><?=$this->icon('collapse-closed') ?> <?=$topLabel ?></a>
```

Comment:
This one doesn't seem to be working.

Reply:
(To test, you need to collapse your browser to a smaller size, and then the top facets become collapsible... looks fine on dev, but is broken on this branch -- icon is totally missing in bootprint3 theme; I didn't check the others).